sdcm/argus_results.py

```diff
@@ -106,6 +113,12 @@ def send_result_to_argus(argus_client: ArgusClient, workload: str, name: str, de
                                     row=f"Cycle #{cycle}",
                                     value=value,
                                     status=Status.PASS if value < operation_error_thresholds[f"percentile_{percentile}"] else Status.ERROR)
+        if value := summary[operation.upper()].get("throughput", None):
+            result_table.add_result(column=f"Throughput {operation}",
```

Comment:
safer would be `{operation.lower()}`, just in case (would raise error and fail sending result).

Reply:
done